assets/blocks/course-outline/edit.js

```diff
@@ -85,8 +98,39 @@ const EditCourseOutlineBlock = ( {
 	);
 };
 
-export default withSelect( ( select ) => {
-	return {
-		structure: select( COURSE_STORE ).getStructure(),
+const applyWithSelect = withSelect( ( select ) => ( {
+	structure: select( COURSE_STORE ).getStructure(),
+} ) );
+
+const applyWithDispatch = withDispatch( ( dispatch, ownProps, { select } ) => {
+	const editPost = select( 'core/edit-post' );
+	const { toggleEditorPanelEnabled } = dispatch( 'core/edit-post' );
```

Comment:
Unfortunately, this action only shows/hides the panel. It continues submitting to the server.